test/insight.js

```diff
@@ -141,7 +141,7 @@ describe('Insight', function() {
     var data = require('./models/sampleAddressFromInsight.json');
```

Comment:
How about making sampleAddressFromInsight itself stringified instead of stringifying it in both places where it's used?